docs/connect.html

```diff
@@ -999,7 +999,7 @@ <h4><a id="connect_schemas" href="#connect_schemas">Working with Schemas</a></h4
 
     <p>Sink connectors are usually simpler because they are consuming data and therefore do not need to create schemas. However, they should take just as much care to validate that the schemas they receive have the expected format. When the schema does not match -- usually indicating the upstream producer is generating invalid data that cannot be correctly translated to the destination system -- sink connectors should throw an exception to indicate this error to the system.</p>
 
-    <h4><a id="connect_administration" href="#connect_administration">Kafka Connect Administration</a></h4>
+    <h3><a id="connect_administration" href="#connect_administration">8.4 Administration</a></h3>
```

Comment:
This clearly doesn't belong under `Connector Development Guide`; I'm guessing it was meant to be a top level heading and have corrected it accordingly.